Tevinthuku/Politico
# [Finishes #163685096] Completed working on LFA feedback

**What does this PR do ?**
This PR moves the CSS file to a different directory and awesome badges to the README. as per the LFA [feedback](https://docs.google.com/document/d/1KLzfvaYD9BMLvw466uSOzgXfZtqHnAKl1JGFAqukp9M/edit) provided

**Tasks to be completed**
:ballot_box_with_check: Move CSS to its own folder
:ballot_box_with_check: Add badges to the README file

**How is this to be manually tested**
1. Clone this repo
2. Checkout to gh-pages
3. See the README 
4. Visit the [site](file:///home/tevin/Documents/projects/Politico/UI/index.html) it still works
5. Check the repo file structure, there is now a CSS folder

![image](https://user-images.githubusercontent.com/12128153/52185575-ead75580-2831-11e9-89a1-334db9abb97c.png)
